olive/evaluator/evaluation.py

```diff
@@ -13,11 +13,13 @@
 from olive.constants import Framework
 from olive.evaluator.accuracy import AUC, AccuracyScore, F1Score, Precision, Recall
 from olive.evaluator.metric import AccuracySubType, LatencySubType, Metric
+from olive.hardware import Device
 from olive.model import OliveModel
-from olive.systems.common import Device
 
 
-def evaluate_accuracy(model: OliveModel, metric: Metric, device: Device = Device.CPU) -> Dict[str, Any]:
+def evaluate_accuracy(
+    model: OliveModel, metric: Metric, device: Device = Device.CPU, execution_providers: Union[str, List[str]] = None
+) -> Dict[str, Any]:
```

Comment:
How about using single AcceleratorSpec argument instead of two separate arguments for Device and EP?

Reply:
The reason that I don't use AcceleratorSpec here is perf tuning would like to pass one device and multiple execution providers if the pass is called explicitly.